config.go

```diff
@@ -125,6 +126,7 @@ type config struct {
 	FreeTxRelayLimit     float64       `long:"limitfreerelay" description:"Limit relay of transactions with no transaction fee to the given amount in thousands of bytes per minute"`
 	Listeners            []string      `long:"listen" description:"Add an interface/port to listen for connections (default all interfaces port: 8333, testnet: 18333)"`
 	LogDir               string        `long:"logdir" description:"Directory to log output."`
+	LogCompressor        string        `long:"logcompressor" description:"Compression algorithm to use when rotating logs.  Valid algorithms are: gzip and zstd."`
```

Comment:
Could add `choice:"gzip" choice:"zstd"` to give some validation responsibility to the config parsing library.

Reply:
nice, didn't realize those were being useful anywhere 😓